packages/utils/src/lib/file-system.ts

```diff
@@ -12,6 +12,7 @@ export async function readTextFile(path: string): Promise<string> {
 
 export async function readJsonFile<T = unknown>(path: string): Promise<T> {
   const text = await readTextFile(path);
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
   return JSON.parse(text);
```

Comment:
This way is maybe a bit nicer:

```suggestion
  return JSON.parse(text) as T;
```